packages/slice-machine/src/models/slice/context.tsx

```diff
@@ -55,14 +49,22 @@ export const SliceHandler: React.FC = ({ children }) => {
     }
   })();
 
-  if (!variation) {
-    replace("/");
-    return null;
+  // TODO: enabling these redirects will break the "A user can create and rename a slice" E2E test.
```

Comment:
Do we lose any other feature by disabling them? Do we have a separate ticket for a fix?